### PR TITLE
refactor: move `status_display_bottom_text` and `fire_alarm_light_color` to security level prototypes

### PR DESCRIPTION
**Pull request by Gaxeer**
## About The Pull Request

Move security level related data from switch-cases to security level prototypes.

## Why It's Good For The Game

Nothing player facing.
Cleaner code for coders

## Changelog

:cl:
refactor: move `status_display_bottom_text` and `fire_alarm_light_color` to security level prototypes
/:cl:

### CODE REVIEW

File: code/modules/security_levels/security_level_datums.dm [hunk 1]
```diff
@@ -13,6 +13,10 @@
 	var/announcement_color = "default"
 	/// The numerical level of this security level, see defines for more information.
 	var/number_level = -1
+	/// Bottom text that will be posted to status displays on this security level
```
Comment (lessthnthree) [inline code/modules/security_levels/security_level_datums.dm:16]:
```suggestion
	/// Icon state that will be displayed on displays during this security level
```

File: code/modules/security_levels/security_level_datums.dm [hunk 2]
```diff
@@ -13,6 +13,10 @@
...
+	var/status_display_bottom_text
```
Comment (lessthnthree) [inline code/modules/security_levels/security_level_datums.dm:17]:
```suggestion
	var/status_display_icon_state
```

File: code/game/machinery/computer/communications.dm [hunk 3]
```diff
@@ -380,15 +380,7 @@
...
-					switch(SSsecurity_level.get_current_level_as_number())
-						if(SEC_LEVEL_DELTA)
-							post_status("alert", "deltaalert")
-						if(SEC_LEVEL_RED)
-							post_status("alert", "redalert")
-						if(SEC_LEVEL_BLUE)
-							post_status("alert", "bluealert")
-						if(SEC_LEVEL_GREEN)
-							post_status("alert", "greenalert")
+					post_status("alert", SSsecurity_level?.current_security_level?.status_display_bottom_text || "greenalert")
```
Comment (lessthnthree) [inline code/game/machinery/computer/communications.dm:383]:
```suggestion
					post_status("alert", SSsecurity_level?.current_security_level?.status_display_icon_state || "greenalert")
```

File: code/modules/modular_computers/file_system/programs/statusdisplay.dm [hunk 4]
```diff
@@ -59,15 +59,7 @@
...
-			switch(SSsecurity_level.get_current_level_as_number())
-				if(SEC_LEVEL_DELTA)
-					post_status("alert", "deltaalert")
-				if(SEC_LEVEL_RED)
-					post_status("alert", "redalert")
-				if(SEC_LEVEL_BLUE)
-					post_status("alert", "bluealert")
-				if(SEC_LEVEL_GREEN)
-					post_status("alert", "greenalert")
+			post_status("alert", SSsecurity_level?.current_security_level?.status_display_bottom_text || "greenalert")
```
Comment (lessthnthree) [inline code/modules/modular_computers/file_system/programs/statusdisplay.dm:62]:
```suggestion
			post_status("alert", SSsecurity_level?.current_security_level?.status_display_icon_state|| "greenalert")
```

File: code/modules/security_levels/security_level_datums.dm [hunk 5]
```diff
@@ -47,6 +51,8 @@
 	announcement_color = "green"
 	sound = 'sound/misc/notice2.ogg' // Friendly beep
 	number_level = SEC_LEVEL_GREEN
+	status_display_bottom_text = "greenalert"
```
Comment (lessthnthree) [inline code/modules/security_levels/security_level_datums.dm:54]:
```suggestion
	status_display_icon_state = "greenalert"
```

File: code/modules/security_levels/security_level_datums.dm [hunk 6]
```diff
@@ -60,6 +66,8 @@
 	announcement_color = "blue"
 	sound = 'sound/misc/notice1.ogg' // Angry alarm
 	number_level = SEC_LEVEL_BLUE
+	status_display_bottom_text = "bluealert"
```
Comment (lessthnthree) [inline code/modules/security_levels/security_level_datums.dm:69]:
```suggestion
	status_display_icon_state = "bluealert"
```

File: code/modules/security_levels/security_level_datums.dm [hunk 7]
```diff
@@ -74,6 +82,8 @@
 	announcement_color = "red"
 	sound = 'sound/misc/notice3.ogg' // More angry alarm
 	number_level = SEC_LEVEL_RED
+	status_display_bottom_text = "redalert"
```
Comment (lessthnthree) [inline code/modules/security_levels/security_level_datums.dm:85]:
```suggestion
	status_display_icon_state = "redalert"
```

File: code/modules/security_levels/security_level_datums.dm [hunk 8]
```diff
@@ -88,5 +98,7 @@
 	announcement_color = "purple"
 	sound = 'sound/misc/airraid.ogg' // Air alarm to signify importance
 	number_level = SEC_LEVEL_DELTA
+	status_display_bottom_text = "deltaalert"
```
Comment (lessthnthree) [inline code/modules/security_levels/security_level_datums.dm:101]:
```suggestion
	status_display_icon_state = "deltaalert"
```

**Review by lessthnthree — CHANGES_REQUESTED** [review]
These are icon states, not strings passed to the display.

**Comment by Gaxeer** [comment]
> These are icon states, not strings passed to the display.

Thanks for review. Applied your adjustments

**Pull request merged**